mantidproject/mantid
# Tomographic Reconstruction Interface - Reset all buttons now ask for confirmation

## Description of work

Added confirmation dialogue for the `Reset All` buttons on the `Filters` and `System` tabs.
## To test:
1. Open `Interfaces` -> `Diffraction` -> `Tomographic Reconstruction`
2. Go to `Filters`/`System` tabs
3. Change some of the settings 
4. Press `Reset all` button at the bottom (it will reset only for the current tab)
5. You should be asked for confirmation by a slightly intimidating QuestionBox.
6. OPTIONAL: If the question box is not intimidating enough, it will be fixed
7. Pressing `No` should **NOT** change anything 
8. Pressing `Yes` should change the settings back to their default values



Fixes #17164 .



_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
